composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require-dev": {
+        "jangregor/phpstan-prophecy": "^1.0"
+    }
+}
```

Comment:
It looks like `composer require` was run at the root of the project which created this file. If we need this package it should be installed in either client or API so we can drop this file.